sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -157,7 +158,9 @@ def test_source_display_data(self):
     # No extra avro parameters for AvroSource.
     expected_items = [
         DisplayDataItemMatcher('compression', 'auto'),
-        DisplayDataItemMatcher('file_pattern', file_name)]
+        DisplayDataItemMatcher(
+            'file_pattern',
+            'StaticValueProvider(type=str, value=\'%s\')' % file_name)]
```

Comment:
The display data should just show the `file_name`, not the whole `StaticValueProvider` object. This should be very easy to fix in the class' `display_data` method in `avroio.py`.